mappings/net/minecraft/entity/mob/ZombieEntity.mapping

```diff
@@ -3,6 +3,8 @@ CLASS net/minecraft/class_1642 net/minecraft/entity/mob/ZombieEntity
 	FIELD field_7424 ticksUntilWaterConversion I
 	FIELD field_7425 CONVERTING_IN_WATER Lnet/minecraft/class_2940;
 	FIELD field_7426 inWaterTime I
+	FIELD field_7427 ZOMBIE_TYPE Lnet/minecraft/class_2940;
+		COMMENT Unused tracked data, left over from 1.10 when zombies, zombie villagers and husks were all the same entity type.
```

Comment:
Did they share the same entity type or were the all of them a `ZombieEntity` just with a flag

Reply:
`EntityType` didn't exist back then in the code, there was only `Class<? extends Entity>`. They were all the same class with just this flag.

Reply:
I'd probably say they were the same entity then in the comment

Reply:
"same entity" would imply to me that they are the same entity instance

Reply:
Maybe "same type of entity"? That'd work around "entity type" being taken while still meaning the same time.